PNL/py/pnl.py

```diff
@@ -24,7 +26,7 @@
 ## TODO configuration file for this value!
 sys.path.append('../../HARK')
 
-AZURE = True
+AZURE = False
```

Comment:
I know this is annoying, but ...

It's better to have Azure "turned on" in version control because for the time being because ultimately we're interested in Azure deployments, and if we, say, build a container with a clone of this repository we wouldn't want to have to script the flipping of all these `AZURE` switches.

Of course the real solution is to remove the switches entirely #6